scripts/apps/search/components/fields/update.tsx

```diff
@@ -1,27 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {gettext} from 'core/ui/components/utils';
 
 export const update: React.StatelessComponent<any> = (props) => {
-    const {$interpolate, gettextCatalog} = props.svc;
-
     if (props.item.correction_sequence) {
         return React.createElement(
             'div',
             {
                 className: 'provider',
                 key: 'update',
-                title: gettextCatalog.getString('correction sequence'),
-            },
-            $interpolate(
-                gettextCatalog.getString('Update {{ seq }}'))({
-                seq: props.item.correction_sequence,
+                title: gettext('correction sequence'),
             },
-            ),
+            gettext('Update {{seq}}', {seq: props.item.correction_sequence}),
```

Comment:
Don't use a shorthand for "the sequence", it makes it hard for a translator to understand the context and choose the correct translation. I know it already was like this, but let's fix it.

Reply:
Done